### PR TITLE
issue #27: add mau

### LINKED ISSUE #27: awg-exporter: Add MAU

### PR DESCRIPTION
**Pull request by DarthSidious007**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,18 @@
-FROM ubuntu:24.04 AS exporter
+FROM ubuntu:24.04
 RUN apt-get update && \
     apt-get install python3-pip -y && \
+    rm -rf /var/lib/apt/lists/* && \
```
Comment (tiaga) [inline Dockerfile:4]: Seems https://github.com/hadolint/hadolint/wiki/DL3009 will be a bit better.

File: docker-compose.base.yml
Comment (tiaga) [inline docker-compose.base.yml:1]: Do we really need all this Compose files?

File: exporter.py [hunk 2]
```diff
@@ -354,7 +362,10 @@ def main_loop(self):
         'grafana_write_url': config('AWG_GRAFANA_WRITE_URL', default=''),
         'grafana_write_token': config('AWG_GRAFANA_WRITE_TOKEN', default=''),
         'grafana_additional_labels': config('AWG_GRAFANA_ADDITIONAL_LABELS', default=''),
-        'awg_executable': config('AWG_EXPORTER_AWG_SHOW_EXEC', default='awg show')
+        'awg_executable': config('AWG_EXPORTER_AWG_SHOW_EXEC', default='awg show all dump'),
+        'redis_host': config('AWG_EXPORTER_REDIS_HOST', default='localhost'), # default='amneziawg-exporter-redis'),
```
Comment (tiaga) [inline exporter.py]: Seems we don't need this comment

File: docker-compose.base.yml
Comment (DarthSidious007) [reply docker-compose.base.yml:1]: removed

File: Dockerfile (hunk 1 above)
Comment (DarthSidious007) [reply Dockerfile:4]: there is no need in our case

**Review by tiaga — APPROVED** [review]

**Pull request merged**